.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: node_js
 node_js:
+  - '6'
   - '5'
```

Comment:
Ok, I don't feel strongly either way. Mostly just annoying having to do this update twice :p